src/Database/TranslationService.php

```diff
@@ -158,7 +158,7 @@ public function findForUpdate(Translatable $model, $language, $field)
             'field' => $field
         ]);
 
-        if (count($translations)) {
+        if (isset($translations[0])) {
```

Comment:
What exactly was the issue here?

Reply:
It was at least the `testTranslationUpdateWithNullTranslation`. In previous php versions `count(null)` was a warning, but now it causes an exception.